_posts/2017-06-27-ardublocklysenseBox.md

```diff
@@ -26,7 +26,16 @@ Grundvorraussetzung für die Verwendung von Ardublockly ist, dass die [Arduino I
 
 Unter [Arbublockly](senseBox.de/blockly) befindet sich eine online Version der Oberfläche die ohne Installation auf dem gängigen Browsern läuft. Hierbei muss allerdings der genierte Code mittels Copy and Paste in die Arduino IDE kopiert werden.
 
-Die offline Varianten der Ardublockly Oberfläche befinden sich im [Github Repository](github.com/senseBox/ardublockly) in den Releases (die Releases werden in unregelmäßigen Abständen neu veröffentlicht). Hier gibt es für die gängigen Betriebssysteme (Windows, MacOS, Linux) eine jeweilige Version.
+Die offline Varianten der Ardublockly Oberfläche befinden sich im [Github Repository](github.com/senseBox/ardublockly) in den [Releases](https://github.com/sensebox/ardublockly/releases)  (die Releases werden in unregelmäßigen Abständen neu veröffentlicht). Hier gibt es für die gängigen Betriebssysteme (Windows, MacOS, Linux) eine jeweilige Version.
 Für die Offline Variante muss Python (mind. Version 2.7) installiert sein. Weitere Informationen und der Download der Software befinden sich auf der [Python Homepgage](https://www.python.org/). Unter MacOS ist Python bereits vorinstalliert. Hierbei muss nur die ardublock.app in den Programmordner kopiert werden.
 
-Damit die Sensoren ausgelesen werden können wird eine Software Bibliothek benötigt, die [senseBox-Library](https://github.com/sensebox/senseBox_library) beeinhaltet die Software für alle Sensoren, die in der senseBox:edu enthalten sind. Die Software Library muss heruntergeladen werden und in die Arduino IDE integriert werden (mehr Informationen dazu [hier](https://edu.books.sensebox.de/de/getting_started/installation_der_software.html))  
+Damit die Sensoren ausgelesen werden können wird eine Software Bibliothek benötigt, die [senseBox-Library](https://github.com/sensebox/senseBox_library) beeinhaltet die Software für alle Sensoren, die in der senseBox:edu enthalten sind. Die Software Library muss heruntergeladen werden und in die Arduino IDE integriert werden (mehr Informationen dazu [hier](https://edu.books.sensebox.de/de/getting_started/installation_der_software.html))
+
+Verwendung der Oberfläche
+============
+Der Screenshot zeigt den Grundaufbau der Ardublockly Oberläche. Im Auswahlmenü rechts können die Programmierblöcke herausgezogen und in der Mitte zusammengesetzt werden. Es empfiehlt sich mit dem Block Arduino Führe zueerst aus und Arduino Endlosschleife aus dem Bereich Funktionen zu starten. Im rechten Fenster wird der generierte Code angezeigt. Hier lässt sich als nachvollziehen welche Programmierbefehle hinter den Blöcken stehen.
```

Comment:
Hier stimmt was nicht.  Ich würde es in Anführungszeichen schreiben, um die Lesbarkeit zu verbessern"
"Arduino Führe zueerst aus"
"Arduino Endlosschleife"

"Hier lässt sich als(o) nachvollziehen welche Programmierbefehle hinter den Blöcken stehen."
Den Satz würde ich noch mal umformulieren